packages/components/src/Banner/Banner.tsx

```diff
@@ -4,18 +4,34 @@ import styles from "./Banner.css";
 import types from "./notificationTypes.css";
 import { Icon, IconColorNames } from "../Icon";
 import { Text } from "../Text";
+import { Button, ButtonProps } from "../Button";
 
 interface BannerProps {
   readonly children: ReactNode;
   readonly type: "notice" | "success" | "warning" | "error";
+  readonly primaryAction?: ButtonProps;
+  /**
+   * Determines the presentation style and some options for the banner. A site
+   * banner is not dismissable and appears at the top of the site layout. A
+   * page banner can be dismissed and appears inline at the top of the page.
+   *
+   * @default "page"
+   */
+  readonly placement?: "page" | "site";
```

Comment:
I like this distinction, but this whole prop might be moot if BizOps just finished building a sitewide banner. We might want to leave this prop out for now...

Reply:
Yeah, the more I think about this, its a really good idea from design. But, we're not there yet in being able to fully implement it. I.e. just because we put this prop on this banner doesn't mean people start using it instead of building their own. We need to address the problem of multiple banners before we implement it here with a prop.